### PR TITLE
Enable OSX

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
@JohanMabille note that I have skipped some `.so` files installation, which are not found on OS X.

File: CMakeLists.txt [hunk 1]
```diff
@@ -196,10 +202,12 @@ if (UNIX)
                 ${openssl_dir}/openssl.pc
           DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/pkgconfig)
 
+if(NOT APPLE)
   install(PROGRAMS ${openssl_dir}/engines/afalg.so
                    ${openssl_dir}/engines/capi.so
                    ${openssl_dir}/engines/padlock.so
           DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/engines-1.1)
+endif()
```
Comment (SylvainCorlay) [inline CMakeLists.txt]: :warning:

**Comment by JohanMabille** [comment]
Yes I saw. DI you check if they are generated in a different folder? Or are you sure they are not generated at all? (just to know if I should check or not).

**Comment by SylvainCorlay** [comment]
> Yes I saw. DI you check if they are generated in a different folder? Or are you sure they are not generated at all? (just to know if I should check or not).

I will locally when I have mac handy.

**Comment by JohanMabille** [comment]
Tests are green so let's merge!

**Pull request merged**